apps/prairielearn/src/lib/externalGraderSqs.js

```diff
@@ -31,8 +31,8 @@ export class ExternalGraderSqs {
 
     async.series(
       [
-        (callback) => {
-          buildDirectory(dir, submission, variant, question, course, callback);
+        async () => {
```

Comment:
Nit: consider collapsing this together with the below `async () => { ... }` block.

Reply:
Done in 2210198a4af704464ceb91c7d5e81465878261c5.